cmd/interceptors/main.go

```diff
@@ -82,26 +79,41 @@ func main() {
 	mux.Handle("/", service)
 	mux.HandleFunc("/ready", handler)
 
-	keyFile, certFile, caCert, err := server.CreateCerts(ctx, kubeClient.CoreV1(), logger)
+	tc, err := triggersclientset.NewForConfig(cfg)
 	if err != nil {
 		return
 	}
 
-	tc, err := triggersclientset.NewForConfig(cfg)
+	serverCert, caCert, err := server.CreateCerts(ctx, kubeclient.Get(ctx).CoreV1(), time.Now().Add(server.Decade), logger)
 	if err != nil {
 		return
 	}
 
-	if err := listAndUpdateClusterInterceptorCRD(ctx, tc, service, caCert); err != nil {
+	if err := service.ListAndUpdateClusterInterceptorCRD(ctx, tc.TriggersV1alpha1(), caCert); err != nil {
 		return
 	}
+
+	// After creating certificates using CreateCerts lets validate validity of created certificates
+	service.CheckCertValidity(ctx, serverCert, caCert, kubeclient.Get(ctx).CoreV1(), logger, tc.TriggersV1alpha1(), time.Minute)
+
+	interceptorSecretName := os.Getenv("INTERCEPTOR_TLS_SECRET_NAME")
```

Comment:
`INTERCEPTOR_TLS_SECRET_NAME` should be under some constant.

Reply:
Done